backend/src/org/commcare/resources/model/ResourceTable.java

```diff
@@ -419,83 +433,64 @@ public void prepareResources(ResourceTable master, CommCareInstance instance)
      * @throws UnfullfilledRequirementsException resource(s) incompatible with
      *                                           current CommCare version
      */
-    public void prepareResources(ResourceTable master,
-                                 CommCareInstance instance,
-                                 String toInitialize)
-            throws UnresolvedResourceException, UnfullfilledRequirementsException {
+    public void prepareResourcesUpTo(ResourceTable master,
```

Comment:
Kind of an awkwardly named/used method. Since it's only getting used once for a very specific purpose (to prepare the table with just the profile), might be good to rename/rewrite to do that explicitly.